src/622/modules/password/index.js

```diff
@@ -0,0 +1,14 @@
+const words = ['acesa', 'ajuda', 'amigo', 'antes', 'arcos', 'baile', 'balas', 'bispo', 'chefe','cheio', 'cinto', 'cravo', 'etapa', 'etnia', 'flora', 'lazer', 'legal', 'lugar', 'parte', 'parto', 'perto', 'porta', 'regra', 'resto', 'salve', 'sente', 'setor', 'sexta', 'tecla', 'tinta', 'torta', 'touro', 'trato', 'valer', 'veado']
```

Comment:
Acho que estes arquivos vieram do outro PR, não?

Quando criar uma branch para uma tarefa nova, o ideal é que se crie sempre a partir da master, para não vir arquivos de outro PR.

Reply:
Corrigido no commit [fix: removing the password module from this branch](https://github.com/dojo-coders/keep-talking-defuser-helper/pull/22/commits/c89b3ce8046845280a2d52320106afb32c832d8e)